### ISSUE TITLE
Release 0.6.1 jar is missing some dependencies

### ISSUE THREAD

**Issue by DavidBiesack** [siom79/japicmp#95]
I downloaded the 0.6.1 release jar from the Releases page and tried running
```
java -jar /u/sasdjb/java/lib/japicmp-0.6.1-jar-with-dependencies.jar -o old.jar -n new.jar
```
and got an error

```
E: Class not found: com.jayway.jsonpath.Configuration$Defaults
See '--help' or '-h' for more information.
```

Perhaps the release jar is not properly assembled.

(P.S. Please add instructions to the README for how to run the tool)

**Comment by siom79** [comment]
It looks like the class `com.jayway.jsonpath.Configuration$Defaults` is missing on your classpath. To get a fast result you can use the command line switch `--ignore-missing-classes`.

The next release should have a more expressive error description.

You can also add additional dependencies to the classpath like you would do for a normal standalone application (see for example [here](http://siom79.github.io/japicmp/CliTool.html)):
```
java -cp japicmp-0.6.1-jar-with-dependencies.jar;otherLibrary.jar japicmp.JApiCmp -n new-version.jar -o old-version.jar
```

**Comment by DavidBiesack** [comment]
Ok, this is reporting a class that *my* jar depends on; I thought perhaps it was a jar that japicmp depends on but which was missing from japicmp-0.6.1-jar-with-dependencies.jar
Using `--ignore-missing-classes` gave me a valid report.

**Comment by siom79** [comment]
I have improved the error message for the next release. I hope this will make it more clear.
```
E: Could not load 'org.apache.commons.math3.stat.inference.TTest'. Please make sure that all libraries have been added to the classpath (CLASSPATH=) or try the option '--ignore-missing-classes'.
See '--help' or '-h' for more information.
```

**Comment by siom79** [comment]
Released with 0.6.2.
